geode-core/src/main/java/org/apache/geode/internal/cache/RegionPerfStats.java

```diff
@@ -313,13 +336,17 @@ public void incConflatedEventsCount() {
 
   @Override
   public void endGet(long start, boolean miss) {
+    long totalNanos = 0;
     if (clock.isEnabled()) {
-      long totalNanos = getTime() - start;
+      totalNanos = getTime() - start;
       stats.incLong(getTimeId, totalNanos);
     }
     stats.incLong(getsId, 1L);
     if (miss) {
       stats.incLong(missesId, 1L);
+      cacheGetsMissTimer.record(totalNanos, NANOSECONDS);
```

Comment:
I thought we were going to measure these in Get70, to include all of the overhead, and to ensure that we are measuring only gets that originated outside the server. Has this changed?

Reply:
> I'm ok with counting gets from REST, GFSH, Functions, etc. Dale has a point about timing the overhead in Get70 but I don't feel strongly about it.

If we do decide to leave the instrumentation as-is, then we should probably add some additional test for "get all", GFSH, REST, etc.

Reply:
Addressed in latest commit